src/battle.ts

```diff
@@ -3355,6 +3359,27 @@ class Battle {
 			this.resetTurnsSinceMoved();
 			let poke = this.getPokemon(args[1])!;
 			let move = Dex.moves.get(args[2]);
+
+			let furyCutterCap = 4;
+			let furyCutterBP = 40;
+			if (this.gen <= 4) {
+				furyCutterCap = 16;
+				furyCutterBP = 10;
+			} else if (this.gen === 5) {
+				furyCutterCap = 8;
+				furyCutterBP = 20;
+			}
+
+			if (move.id === 'furycutter') {
+				if (!poke.hasVolatile('furycutter' as ID)) {
+					poke.addVolatile('furycutter' as ID, 2, furyCutterBP);
+				} else if (poke.volatiles['furycutter'][1] < furyCutterCap) {
+					poke.volatiles['furycutter'][1] *= 2;
```

Comment:
```suggestion
					poke.volatiles['furycutter'][1] *= 2;
					poke.sprite.updateStatbar(poke);
```

Reply:
(This is why the statbar isn't updating.)

Reply:
This did not work unfortunately

Reply:
My plan right now is to possibly add and remove the fury cutter volatile when appropriate, in the way that the 'perish' volatiles refresh their status bar message by doing so. Except that it would be copies of the same volatile and not 4 different types of one.